anitya/tests/base.py

```diff
@@ -287,5 +288,23 @@ def create_flagged_project(session):
     return flag
 
 
+def create_user(session):
```

Comment:
With tests I think explicit is preferable to DRY. Having a function to create a user is indirection that makes the test harder to read an understand at a glance. I don't think this saves a huge amount of typing and isn't worth that indirection.

Reply:
I'm not sure if this is true, the user is created on many places in tests. When I added this feature, almost 50 tests started failing. So this saved me a lot typing.

Reply:
You are probably right in this, it is better if this is done in the same place as tests, not in imported code.